target/linux/generic/files/drivers/platform/mikrotik/rb_nvmem.c

```diff
@@ -0,0 +1,214 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
```

Comment:
Quick nit, missing an SPDX-FileCopyrightText here (or a copyright mention in code comment), AIUI "Module Author" isn't enough.